_aggregations/pipeline/derivative.md

```diff
@@ -0,0 +1,221 @@
+---
+layout: default
+title: Derivative
+parent: Pipeline aggregations
+nav_order: 70
+redirect_from:
```

Comment:
```suggestion
```

Reply:
Redirects are not needed because they don't work for specific headings and there is currently no page for derivative aggregations.